### PR TITLE
Test setup

### PR DESCRIPTION
**Pull request by shli-wish**
- added two packages for demo: add-button with successful e2e and unit tests, sub-button with failed e2e and unit tests
- test related scripts and instructions are in monorepo/test
- update test commands in wps-button-template-test to fit with the other packages

### CODE REVIEW

File: monorepo/packages/wps-button-template-test/package.json [hunk 1]
```diff
@@ -50,8 +50,9 @@
     "lint": "eslint '*/**/*.{tsx,ts,js}' --fix",
     "prettier": "prettier --write './**/*.{tsx,ts,js}'",
     "test:unit": "jest --testPathIgnorePatterns dist/*",
-    "test:e2e-local": "nightwatch --env chrome",
-    "test:e2e": "nightwatch --env chromeMD"
+    "test:e2e": "nightwatch --env chrome",
+    "test:e2e:md": "nightwatch --env chromeMD",
+    "test:e2e:bg": "bash ../../test/nightwatch_runner.sh"
```
[the remark this answers is not in the document]
Comment (shli-wish) [reply monorepo/packages/wps-button-template-test/package.json:55]: md: merchant dashboard - e2e test with merchant dashboard server
bg: background - e2e test with dev server running in background
Not sure if it is too long if we put in full word.

File: monorepo/packages/wps-button/node_modules/.bin/tsc [hunk 2]
```diff
@@ -1 +0,0 @@
-../../../../node_modules/typescript/bin/tsc
```
[the remark this answers is not in the document]
Comment (shli-wish) [reply monorepo/packages/wps-button/node_modules/.bin/tsc:1]: I think we forgot to add .gitignore initially so node_modules were included. And I guess these two are the leftover ones after the cleanup.

File: monorepo/packages/wps-button-template-test/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (shli-wish) [reply monorepo/packages/wps-button-template-test/package.json:55]: Do the following names make sense?
test:e2e:merchant-dashboard
test:e2e:dev-server-background

[the remark this answers is not in the document]
Comment (ouding-wish) [reply monorepo/packages/wps-button-template-test/package.json:55]: Ah, I see. I think MD can stay as `test:e2e:md` then, but bg could be changed to `test:e2e:dev` (testing in a dev environment). I think that gives more context than "bg"

**Pull request merged**